constants_test.py

```diff
@@ -15,18 +15,19 @@
 """Tests for Constants object and cosntants.json file."""
 
 import os
-import json
 
+import constants #pylint: disable=relative-import
 from core.tests import test_utils #pylint: disable=relative-import
 
 class ConstantsTests(test_utils.GenericTestBase):
 
     def test_constants_file_is_existing(self):
         """Test if the constants file is existing."""
         self.assertTrue(os.path.isfile(os.path.join(
-            'assets', 'constants.json')))
+            'assets', 'constants.js')))
 
-    def test_constants_file_is_valid_json(self):
+    def test_constants_file_contains_valid_json(self):
         """Test if the constants file is valid json file."""
-        with open(os.path.join('assets', 'constants.json'), 'r') as f:
-            self.assertTrue(isinstance(json.load(f), dict))
+        with open(os.path.join('assets', 'constants.js'), 'r') as f:
+            json = constants.parse_json(f)
+            self.assertTrue(isinstance(json, dict))
```

Comment:
I think it would be best to add constant just for this. Because when I would pick some random constant and someone would later change it, the test would fail even when they did nothing wrong.